jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/NamedObjectItemPresenter.java

```diff
@@ -110,6 +128,12 @@ public void signalParameterAddedOrRemoved() {
 
         void setValue(final String value);
 
+        String getName();
```

Comment:
I think you will find that these gettter methods arent needed once you can the presenter passing the new value.